src/app/static/src/app/components/header/DataExplorationSupportMenu.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div id="divclass">
+        <drop-down text="support" :right="true" :delay="true" style="flex: none">
+            <a class="dropdown-item"
+               v-translate="'reportIssues'">
```

Comment:
What was the reasoning behind having a different title for this to the other support menu (that one says "Troubleshooting request" rather than "Report issues")? I presume you're following the mockups here, but reckon we should keep this language consistent one way or the other - maybe one to run past Rachel or Rob to pin down the preferred terminology?

Reply:
They are both changing to Report issues, should update RootState menu.

Reply:
Wasn't it the other way round? "Report issues" got renamed to "Troubleshooting request" in this ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2731 from November.

Reply:
Confusing, I will use "Troubleshooting request" for both.